teuthology/orchestra/remote.py

```diff
@@ -276,11 +277,20 @@ def write_file(self, path, data, sudo=False, mode=None, owner=None,
         :param owner:   set file owner if provided
         :param mkdir:   preliminary create the file directory, defaults False
         :param append:  append data to the file, defaults False
+        :param bs:      block size
```

Comment:
Please, add info about default value.

Reply:
Also give user more idea about what is the block size how it can be used. Probably this will require add some description above.